lib/dogma/rules/final_newline.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dogma.Rules.FinalNewline do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
+  def test(script), do: test(script, [])
+
+  def test(script, _config = []) do
```

Comment:
@lpil getting rid of test/1 is quite a nice idea. Planning on doing some work this weekend so I'll take a look then.